### PR TITLE
added frontend script and ucm field

### PR DESCRIPTION
**Pull request by priyankaSahutekdi**

### CODE REVIEW

File: automation/pom.xml [hunk 1]
```diff
@@ -84,14 +84,22 @@
 			<artifactId>extentreports</artifactId>
 			<version>3.1.5</version>
 		</dependency>
+
```
Comment (priyankaSahutekdi) [inline automation/pom.xml:87]: my test review

Comment (priyankaSahutekdi) [inline automation/pom.xml:87]: Test review

File: automation/src/main/java/com/ucm/pageobjects/DeleteucmtypePage.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+/**
+ * This is Page Class for deleting the ucm data . It contains all the elements and actions
+ * related to deleting ucm data.
+ * 
+ */
+
+public class DeleteucmtypePage extends BaseClass {
+
+	private WebDriver driver;
```
Comment (Kaustubharas) [inline automation/src/main/java/com/ucm/pageobjects/DeleteucmtypePage.java]: @priyankaSahutekdi - Why driver variable is needed in this class? Also it there a need to extend BaseClass here? As I don't see any browser interactions in this class.

Comment (priyankaSahutekdi) [reply automation/src/main/java/com/ucm/pageobjects/DeleteucmtypePage.java]: BaseClass is needed as using logger to show in the report, webDriver is not needed in this class will remove it

**Review by Kaustubharas — APPROVED** [review]
reviewed

**Pull request merged**